addons/website_event_track_quiz/models/event_track.py

```diff
@@ -8,7 +8,7 @@
 class EventTrack(models.Model):
     _inherit = ['event.track']
 
-    quiz_id = fields.Many2one('event.quiz', string="Quiz", compute='_compute_quiz_id', store=True, groups="event.group_event_user")
+    quiz_id = fields.Many2one('event.quiz', string="Quiz", compute='_compute_quiz_id', store=True, groups="event.group_event_registration_desk")
```

Comment:
@minhthuy145 This is not the root of the problem
You must add group `event.group_event_user` to the buttons `action_add_quiz` and `action_view_quiz`

Reply:
> This is not the root of the problem
> You must add group `event.group_event_user` to the buttons `action_add_quiz` and `action_view_quiz`

Okay! let me watch it again. Thank you